src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -213,14 +212,6 @@ public void sanitizeForSaving() {
         this.courseId = Sanitizer.sanitizeForHtml(courseId);
         this.giverEmail = Sanitizer.sanitizeForHtml(giverEmail);
 
-        if (recipients != null) {
```

Comment:
Yes, we need another script for `CommentAttributes` too. To be safe, perhaps I will write a script to check for `recipients` of `CommentAttributes` in database.